lib/dao.js

```diff
@@ -2426,7 +2470,19 @@ DataAccessObject.prototype.updateAttributes = function updateAttributes(data, op
     inst.setAttributes(data);
 
     inst.isValid(function (valid) {
-      if (!valid) {
+
```

Comment:
guess i did something wrong here, changed it again, but now it works. isValid is now only called when needed